### PR TITLE
Update Aspire to 8.0.0-preview.2.23611.9

### PR DESCRIPTION
**Pull request by captainsafia**
This PR relies on non-nuget.org sources at the moment.

### CODE REVIEW

**Comment by captainsafia** [comment]
When running this locally on my macOS, I'm observing the following behavior:

Application host starts up successfully:

```
dotnet run --project src/eShop.AppHost/eShop.AppHost.csproj 
Building...
info: Aspire.Dashboard.DashboardWebApplication[0]
      Now listening on: http://localhost:18848
info: Aspire.Dashboard.DashboardWebApplication[0]
      OTLP server running at: http://localhost:16119
info: Microsoft.Hosting.Lifetime[0]
      Application started. Press Ctrl+C to shut down.
info: Microsoft.Hosting.Lifetime[0]
      Hosting environment: Development
info: Microsoft.Hosting.Lifetime[0]
      Content root path: /Users/captainsafia/gh/eShop/src/eShop.AppHost
```

Docker containers successfully launch:

image

Containers are reported as stuck on `Starting...`

image

And no logs are present in the dashboard. `postgres` says it failed to initialize even though the Docker container is successfully running:

image


Also, not all the projects launch.

**Review by captainsafia — COMMENTED** [review]
✅  Can't approve a PR I opened but this builds on my machine and I was able to complete the purchase flow successfully.

File: nuget.config [hunk 1]
```diff
@@ -4,5 +4,18 @@
     <!--To inherit the global NuGet package sources remove the <clear/> line below -->
     <clear />
     <add key="nuget" value="https://api.nuget.org/v3/index.json" />
+    <add key="dotnet-tools" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json" />
```
Comment (davidfowl) [inline nuget.config:7]:
```suggestion
```

**Review by davidfowl — APPROVED** [review]
This should be good now

**Comment by davidfowl** [comment]
Lets clean up the nuget feed.

**Pull request merged**